### DISCUSSION TITLE
How can I change properties in State?

### DISCUSSION THREAD

**Discussion by qrczak** [spatie/laravel-livewire-wizard#107] [Q&A]
In my state I have boolean property `$doAction` like this:
```
class CartState extends State
{
    public bool $doAction = false;

    public function customerInfo(): array
    {
        $customerInfoState = $this->forStep('customer-info');

        return [
            'name' => $customerInfoState['name'],
            'email' => $customerInfoState['email']
        ];
    }
}
```
and of course it's accessible from StateComponents via `$this->state()->doAction` but when I try change this property it's not working.
The value of this property can be change in each StateComponent depend what radio button you choose that's why I don't want put it every StepComponent.
And if it impossible to change this property from StateComponent how you will solve this problem in other way?
In other words if there are properties common for each StepComponents with possibility modification of them how to do that?
